app/jobs/import_data_from_rapids_job.rb

```diff
@@ -0,0 +1,51 @@
+class ImportDataFromRAPIDSJob < ApplicationJob
+  queue_as :default
+
+  PER_PAGE_SIZE = 50
+
+  def perform
+    service = RAPIDS::API.call
+    start_index = 1
+
+    loop do
+      Rails.logger.info "Processing batchSize: #{PER_PAGE_SIZE}, startIndex: #{start_index}"
+      response = service.work_processes(
+        batchSize: PER_PAGE_SIZE,
+        startIndex: start_index
+      )
+      parsed_response = response.parsed
+      occupation_standards = process_api_response(parsed_response)
+      occupation_standards.map(&:save)
+      total_records = parsed_response["totalCount"]
+      start_index += PER_PAGE_SIZE
+
+      break if start_index > total_records
```

Comment:
What about using a `while` loop instead of breaking out of this loop?

```
while start_index <= total_records
  ...
  start_index += PER_PAGE_SIZE
end
```

Reply:
@littleforest total records comes from the API response. We will need to to a first call outside the while loop to fetch the number of records

Reply:
Ah I see, my suggestion will definitely not work then.